### ISSUE TITLE
use_modular_headers!

### ISSUE THREAD

**Issue by iwheelbuy** [amplitude/Amplitude-iOS#192]
I created a blank project with Xcode 10.2 beta 4 and added amplitude as a dependency using cocoapods:
```
platform :ios, '10.0'

use_modular_headers!

target 'Test' do
    pod 'Amplitude-iOS', '4.6.0'
end
```

And got an error while building the project:

> '...Amplitude_iOS/Amplitude-iOS.modulemap' not found

If I remove ```use_modular_headers!``` from Podfile - everything works. Why?

**Comment by iwheelbuy** [comment]
It was a cocoapods bug.

**Comment by wangchou** [comment]
@iwheelbuy 
I just met the same issue on Cocoapods 1.7.3 & Amplitude-iOS 4.6.0
Did you found a solution for it? Which issue should I track in cocoapods repo?

**Comment by iwheelbuy** [comment]
I believe It was fixed in 1.7.1: [issue](https://github.com/CocoaPods/CocoaPods/issues/8836)
> Fix a 1.7.0 regression in header directory paths when using static libraries

I've just checked. I'm on 1.7.4 Cocoapods & 'Amplitude-iOS', '4.6.0'.
Try cleaning the derived data and etc... It should work!
